Implemented PoyntingVector module

ojha-aditya opened the pull request:
Hereby I have written the functions to implement the Poynting vector for plane polarized electromagnetic waves. The PR contains the following in the PoyntingVectorModule folder:

1. Module PoyntingVector.py which implements the functions to generate the electromagnetic wave expressions and Poynting vector expressions and calculate the numerical value for the same.
2. TestPoyntingVector.py which is the unit test module for PoyntingVector.py
3. Markdown file contains the Physics of Poynting vector calculations done using the above module.
4. There is a separate Jupyter notebook that can be used the test the functions individually.

laserlab commented:
@Tolani4 Please review asap

ojha-aditya commented on PoyntingVectorModule/PoyntingVector.py:
```diff
@@ -0,0 +1,88 @@
+    print("Poynting Vector (in W/m^2): ", S, " = ", Snum)
+    print("The magnitude of Poynting Vector (in W/m^2): ", Smag)
+
+if __name__ == "__main__":
```
So I should remove the main() function loop basically from the module?

There is no column.py in my PR though.

ojha-aditya commented on PoyntingVectorModule/PoyntingVector.py:
```diff
@@ -0,0 +1,88 @@
+    # calculating the Electric, Magnetic Field and Poynting vector
+    E = electric_field_expression(E_0, [kx, ky, kz], delta, f, [nx, ny, nz]) 
+    B = magnetic_field_expression(E_0, [kx, ky, kz], delta, f, [nx, ny, nz])
+    S = poynting_vector_expression(E, B)
```
How do I modify it then? I am sorry I didn't quite understand

ojha-aditya commented:
Also, should I just remove the Jupyter notebook from this PR as it is not longer needed? I had kept it just for trial run purposes.

laserlab commented:
> Also, should I just remove the Jupyter notebook from this PR as it is not longer needed? I had kept it just for trial run purposes.

sure remove it if it is not needed

ojha-aditya commented:
Done. Removed the Jupyter notebook, will soon commit. Also, is there a way to reflect the lint test results in our codespaces so it could be compared and fixed line by line?

ojha-aditya commented:
@laserlab I have comitted all changes requested and got the linting score up as well. Removed the main function and loop, the remaining linting suggestions are a tradeoff it seems, resolving them causes another issue to pop up.

laserlab commented:
merging with wrong formatted docstrings and linting issues
